modules/ob-installing-builds-using-CLI.adoc

```diff
@@ -37,4 +37,36 @@ spec:
 $ oc apply -f sub.yml
 ----
 +
-The {builds-operator} is now installed in the default target namespace `openshift-operators`.
+The {builds-operator} is now installed in the default target namespace `openshift-builds`.
+
+.Verification
+
+After installing the Builds for Red Hat OpenShift Operator, you must verify that the following pods and resources are created to ensure proper functioning of the operator: 
```

Comment:
- Throughout the documentation, B is capital for builds. 
- Similar, attribute isn't added for Red Hat OpenShift Operator in other sections

I'll check both the things with the team. In case we need to update it, I'll consider it under content improvement activity and update it throughout the documentation.